### DISCUSSION TITLE
欸，1Panel终究是走上了商业化。。。

### DISCUSSION THREAD

**Discussion by Starry-Sky-World** [1Panel-dev/1Panel#4690] [Show and tell]
😭

**Comment by wanghe-fit2cloud** [comment]
是的，1Panel 目前已推出专业版功能。
如果您不需要这些功能，可以在面板设置中关闭专业版功能菜单，使用体验与社区版本完全一致。
此外，专业版的推出将促使 1Panel 团队不断开发出更多优质功能，从而进一步提升用户体验。

参考我司另外一款开源项目 [JumpServer](https://github.com/jumpserver/jumpserver)：
![image](https://github.com/1Panel-dev/1Panel/assets/46734380/9ee914de-7276-4cfb-a872-b0fa471fc947)

**Reply by Starry-Sky-World** [reply]
针对中学生/小学生有优惠吗？

**Comment by mobeicanyue** [comment]
商业化才能赚钱，那些因为入不敷出的停更开源项目比比皆是。只要源码保持开源没什么不好的

**Comment by QiQiBiBi** [comment]
没钱赚的项目我才怕……谁敢用啊哈哈哈，搭站可是心血。

**Comment by ynqjwsm** [comment]
已经买了两套支持，希望增加人手，加快新功能迭代
